itou/static/js/matomo.js

```diff
@@ -0,0 +1,16 @@
+$(document).ready(() => {
+
+    /********************************************************************
+       Send an event to Matomo on click.
+       Usage:
+       <a href="#" class="matomo-event" data-matomo-category="MyCategory"
+       data-matomo-action="MyAction" data-matomo-option="MyOption" >
+    ********************************************************************/
+    $("a.matomo-event").on("click", function() {
+        var category = $(this).data("matomo-category");
+        var action = $(this).data("matomo-action");
+        var option = $(this).data("matomo-option");
+
+        _paq.push(['trackEvent', category, action, option]);
+    });
+});
```

Comment:
Un petit retour chariot svp :)

https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
Désolée ! Mais j'ai compris maintenant grâce à ces explications. Je serai plus vigilante à l'avenir.